mewarpx/mewarpx/mwxrun.py

```diff
@@ -302,6 +302,10 @@ def step(self, sim_control, interval=None):
         self.step_interval = compute_step(self.simulation, interval)
         while sim_control.check_criteria():
             self.simulation.step(self.step_interval)
+        # create file to signal that simulation ran to completion
+        if self.me == 0:
+            with open("diags/results.txt") as results_file:
```

Comment:
```suggestion
            with open("diags/results.txt", 'a') as results_file:
```
I get `FileNotFoundError: [Errno 2] No such file or directory: 'results.txt'` without a mode. Append mode seems safer than write as it won't interfere with existing files.